filament/src/RendererUtils.cpp

```diff
@@ -150,7 +151,17 @@ FrameGraphId<FrameGraphTexture> RendererUtils::colorPass(
                 data.depth = builder.read(data.depth, FrameGraphTexture::Usage::DEPTH_ATTACHMENT);
 
                 data.color = builder.write(data.color, FrameGraphTexture::Usage::COLOR_ATTACHMENT);
-                data.depth = builder.write(data.depth, FrameGraphTexture::Usage::DEPTH_ATTACHMENT);
+                if (engine.getConfig().stereoscopicType == StereoscopicType::MULTIVIEW) {
+                    // Add sampleable usage flag for depth in multiview rendering, othewise it's
+                    // treated renderbuffer in the backend and crashed.
+                    data.depth = builder.write(data.depth,
+                        FrameGraphTexture::Usage::DEPTH_ATTACHMENT |
+                        FrameGraphTexture::Usage::SAMPLEABLE);
```

Comment:
this is fine for now -- but I think this is needed because of a bug on OpenGLDriver.cpp.  Can you please file a bug for me?  The issue is that if a texture has more than one layer, we shouldn't use a renderbuffer even if it doesn't have the sampleable usage flag.

Reply:
Yeah, I created a bug for this.  b/329491941